nativescript-angular/index.ts

```diff
@@ -1,7 +1,30 @@
+import "application";
+
+export * from "./platform-common";
 export * from "./platform";
 export * from "./platform-static";
 export * from "./router";
 export * from "./forms";
 export * from "./http";
 export * from "./directives";
 export * from "./common/detached-loader";
+export * from "./trace";
+export * from "./platform-providers";
+export * from "./file-system/ns-file-system";
+export * from "./modal-dialog";
+export * from "./renderer";
+export * from "./view-util";
+export * from "./animation-driver";
+export * from "./resource-loader";
+export * from "./element-registry";
+export * from "./lang-facade";
+export * from "./collection-facade";
+export * from "./router-url-tree";
+export * from "./router/ns-platform-location";
+export * from "./router/ns-router-link-active";
+export * from "./router/ns-router-link";
+export * from "./router/page-router-outlet";
+export * from "./router/router-extensions";
+export * from "./parse5_adapter";
```

Comment:
We should not export parse5_adapter, private_import* as public API here. Maybe we can create a separate "private API" rollup entrypoint?